### ISSUE TITLE
Tea command is broken due to GroupMe not liking spaces

### ISSUE THREAD

**Issue by ErikBoesen** [ErikBoesen/Yalebot#26]

**Comment by victor-hugo-dc** [comment]
I strongly dislike GroupMe spacing. I don't really know if this will work but it doesn't hurt to try I guess? I've noticed that when the lines have words in it they seem to correct the spacing issue.
```msg1 = '{0: <37}'.format(query[0:37])
msg2 = '{0: <30}'.format(query[37:67])
msg3 = '{0: <27}'.format(query[67:94])
msg4 = '{0: <24}'.format(query[94:118])
msg5 = '{0: <21}'.format(query[118:139])

tea = '''
today's tea:
                   )
              (         )
          )  )         (
      )  (   )      (
_(___(____)____(___(____)___
\    {str1}    /___
 \    {str2}    /    |
   \    {str3}    /_____|
     \   {str4}    /
       \    {str5}    /
         \________________/
'''.format(str1=msg1, str2 = msg2, str3 = msg3, str4 = msg4, str5 = msg5)

**Comment by ErikBoesen** [comment]
@victor-hugo-dc I just fixed this issue (mostly) this morning. I used the > character to force GroupMe to interpret the message as a quote even on mobile. Then, rather than allowing GroupMe to strip spaces off the starting of the line, I used an unusual unicode character which looks like a space but isn't the same character—one of these, I think http://jkorpela.fi/chars/spaces.html

The quote formatting uses consistent monospacing on the web application, but not on mobile, for some reason. It looks pretty close now, and past that I don't think there's much we can do because line widths will vary depending on various factors.
